packages/graphback-core/src/db/dataMapper.ts

```diff
@@ -1,55 +1,36 @@
-import { ModelTableMapping } from './createModelTableMappings';
+import { ModelTableMap } from './buildModelTableMap';
 
-export interface DataMapping {
-    id?: TableID
+export interface TableDataMap {
+    idField?: TableID
     table?: string
     data?: any
     fieldMap?: any
 }
 
 export interface TableID {
     name: string
-    value: any
+    value?: any
 }
 
-export const mapDataToTable = (data: any, modelMap: ModelTableMapping): DataMapping => {
-    const fieldMap = modelMap.fieldMap;
+export const getDatabaseArguments = (modelMap: ModelTableMap, data?: any, fieldMap?: any): TableDataMap => {
+    const idField = modelMap.idField;
 
-    const dataMap: DataMapping = {
-        table: modelMap.tableName,
-        fieldMap
-    }
-
-    if (!fieldMap) {
-        return dataMap;
-    }
+    // TODO: Map fields to custom db names
```

Comment:
What this means. Is this what we do here?

Reply:
This PR is about mapping custom ID fields. In future we want to map custom data fields with the `@db.name` annotation.